docs/src/pages/basics/guide-react/index.md

```diff
@@ -29,12 +29,13 @@ First of all, you need to add `@storybook/react` to your project. To do that, si
 npm i --save-dev @storybook/react
 ```
 
-## Add react and react-dom
+## Add react, react-dom, and babel-core
 
-Make sure that you have `react` and `react-dom` in your dependencies as well because we list these as a peerDependency:
+Make sure that you have `react`, `react-dom`, and `babel-core` in your dependencies as well because we list these as a peerDependency:
```

Comment:
I think this might read better as:
```
Make sure that you have `react`, `react-dom` in your dependencies and and `babel-core` in your devDependencies, because we list these as a peerDependency:
```

Reply:
If one has `babel-core` in their dependencies, it will work as well

Reply:
yes, but it reads as if babel-core needs to be an actual dependency, I guess I am too literal with the word `dependencies` 🤷‍♂️